README.md

```diff
@@ -141,6 +141,18 @@ Decide how many nodes you want to run.
 
 TODO
 
+### Exposing elasticsearch service with a route
+
+Obtain the CA cert from Elasticsearch.
```

Comment:
@nhosoi Is the CA cert installed automatically or does the user need to download the CA cert somehow?

Reply:
@mburke5678, the CA cert is supposed to be installed when the elasticsearch and other containers were installed.  We just need to retrieve the existing one.

Please see Rich's comment.  We may want to change the file name "admin-ca" to make it clearer...  Do you have any recommendation?
https://github.com/openshift/cluster-logging-operator/pull/104
> I think this is the CA cert for the CA that issued the Elasticsearch server cert. admin-ca is slightly misleading - we are assuming that the same CA was used to issue the ES admin cert and the ES http(s) server cert.